ext/drmgr/drmgr.h

```diff
@@ -1216,6 +1216,17 @@ drmgr_register_signal_event_ex(dr_signal_action_t (*func)(void *drcontext,
                                                           dr_siginfo_t *siginfo),
                                drmgr_priority_t *priority);
 
+DR_EXPORT
+/**
+ * Registers a callback function for the signal event, with supplied
+ * user data.
```

Comment:
Please add some kind of link to drmgr_register_signal_event_ex() so that someone looking at the doxygen html will have it.

Reply:
Done. I also added links to other variants  functions related to user_data. Let me know if you want me to remove them.

Reply:
Extra links are good.  I'm wondering if you deliberately made the docs for these new signal funcions shorter than the others?  E.g., drmgr_register_module_load_event_user_data talks about "behaves just like DR's module unload event dr_register_module_unload_event()" and "Allows for the passing of user data".  I would vote for either duplicating the drmgr_register_signal_event_ex comments or changing the doc to not just say "See also" but to be something stronger like "Behaves just like xxx with the addition of a user_data parameter" or something.  WDYT?  Am I being too picky?